bin/yamllintrules

```diff
@@ -5,6 +5,7 @@ ignore: |
   /tests/e2e/tests/pilot/testdata/networking/v1alpha3/rule-503test-destinationrule-c-add-subset.yaml
   /tests/e2e/tests/pilot/testdata/networking/v1alpha3/rule-503test-destinationrule-c-del-subset.yaml
   /tests/e2e/tests/pilot/testdata/networking/v1alpha3/destination-rule-c-private.yaml
+  /install/kubernetes/helm/istio-init/files/crd*.yaml
```

Comment:
can you either remove from this list or justify why it is here?

Reply:
yes, this is a temp line here. i will take care of it once it is ready to review.